bin/snare

```diff
@@ -147,15 +152,20 @@ if __name__ == '__main__':
     parser.add_argument("--update-timeout", help="update snare every timeout ", default='24H')
     parser.add_argument("--server-header", help="set server-header", default='nginx/1.3.8')
     parser.add_argument("--no-dorks", help="disable the use of dorks", type=str_to_bool,  default=True)
-    parser.add_argument("--log-dir", help="path to directory of the log file", default='/opt/snare/')
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False, default='/opt/')
```

Comment:
For me now this parameter a little bit misleading, when I set directory by myself, I set the final directory I want to use and snare creates additional directories. What do you think about it? Should we set the resulting directory or parent directory for the pages?

Reply:
We should probably change this argument''s function to tell the base directory for snare. What do you think?